docs/content/en/docs/getting-started/_index.md

```diff
@@ -76,12 +76,17 @@ spec:
   keptnAppCreationRequestTimeoutSeconds: 30
 ```
 
-Apply the file and restart Keptn to pick up the new config:
+Apply the file and wait for Keptn to pick up the new config:
 
 ```shell
 kubectl apply -f collectorconfig.yaml
 ```
 
+By default, Keptn checks this configuration every 30 seconds.
```

Comment:
I am not sure about this, where did you get this information ?

Reply:
"By default, Keptn checks this configuration every 30 seconds." and if true, how would I configure it (if I wanted to)?

Reply:
In general, once a CR is updated, the Kubernetes API server immediately signals the change to registered watchers. In the case of the KeptnConfig controller this should be the same, we introduce a delay of 10 seconds IF and ONLY IF the controller encounters an error while resolving the new configuration https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/controllers/options/keptnconfig_controller.go#L85C3-L85C3

Reply:
So I would get rid of this sentence and maybe have something like: 
```suggestion
By default, Keptn reacts immediately to a configuration change. Of course, the speed depends on the Kubernetes API server signaling updates and can be influenced by network latency.
```

Reply:
This is wrong. The 30s is the timeout for waiting for all manifest to be applied to your K8s cluster. It does **NOT** impact the interval used to see changes in the configuration.